election_reminder.py

```diff
@@ -1,34 +1,45 @@
+#! /usr/bin/env python
 # -*- coding: utf-8 -*-
+# vim: set noexpandtab ts=2:
 
-import datetime
+import json
 import os
 import requests
 
-contests = [
-	{
-    "contest":"Iowa Caucuses",
-	"date": datetime.date(2020,02,03),
-	"verb": "are",
-	"facts": []
-	},
-]
+import dateutil.parser
+import dateutil.utils
+import dateutil.tz
+
+token = os.getenv("SLACK_TOKEN")
+
+contests = [ ]
+
+with open('data.json') as f:
+	contests = json.load(f)
+
 text = None
 for contest in contests:
-	d = contest['date'] - datetime.date.today()
+	dflt_tz = dateutil.tz.tzoffset("EST", -18000)
+	today = dateutil.utils.today(dflt_tz)
+	d = dateutil.parser.parse(contest['date']) - today
 	if d.days < 0:
-		continue #go to next contest
+			continue # go to next contest
 	try:
-		fact = contest['facts'][d.days]
+			fact = contest['facts'][d.days]
```

Comment:
I think something happened to the indentation here.

Reply:
Fixed